src/cli/update.rs

```diff
@@ -622,3 +638,147 @@ impl LockFileDiff {
         .collect()
     }
 }
+
+#[derive(Serialize, Clone)]
+pub struct JsonPackageDiff {
+    name: String,
+    before: Option<JsonChange>,
+    after: Option<JsonChange>,
+    #[serde(rename = "type")]
+    ty: JsonPackageType,
+    #[serde(skip_serializing_if = "std::ops::Not::not")]
+    explicit: bool,
+}
+
+#[derive(Serialize, Clone)]
+pub struct JsonChange {
+    version: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    build: Option<String>,
```

Comment:
There are more things that can change. 

`channel` is an important one I believe, but why not print the full record, before and after? Then the user can fetch what he requires.

Reply:
I wanted to keep it in line with the human output but thats not a bad idea.

Reply:
Sounds good to me 👍🏻